app/controllers/orders_controller.rb

```diff
@@ -11,7 +21,7 @@ def index
         @products = Product.where(user_id: params[:id])
         p @products
         @order_items = OrderItem.where(product: @products, order: @orders)
```

Comment:
Are all of the puts statements in there for debugging? It seems extraneous in terms of readability.